streaming/base/dataset.py

```diff
@@ -43,12 +43,12 @@ class _ShardState(IntEnum):
 
     Restrictions:
     - The initial state of INVALID must be zero.
-    - State transitions: MISSING -> DOWNLOADING -> PRESENT -> MISSING.
+    - State transitions: REMOTE -> DOWNLOADING -> LOCAL -> REMOTE.
     """
     INVALID = 0
```

Comment:
Can you please add a one line description for each what each enum means ?

Reply:
done

Reply:
Wait, I did not see a new commit !!

Reply:
pushed sorry